src/Player.ts

```diff
@@ -0,0 +1,46 @@
+import { Herd } from './Herd';
+
+class Player {
```

Comment:
Please export class Player.

Reply:
Yeah, my bad. The PR got merged 2 days ago so I already know that and I'm going to fix it with the nearest PR unless someone who needs to use it will do it first :)